src/client/Navbar.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useState } from "react";
+import "./App.css";
+import "./styles/Navbar.css";
+import { Link } from "react-router-dom";
+import ReorderIcon from "@mui/icons-material/Reorder";
+import RestaurantIcon from "@mui/icons-material/Restaurant";
+import "./styles/Navbar.css";
+
+
+function Navbar() {
```

Comment:
Good job 💪🏻 
and Thank you for helping I was searching today for this component 😄

Reply:
Hi Islam thanks for the time that you put to review my app:)🙏🏻 glad to help😆